src/TraceEvent/Computers/TraceManagedProcess.cs

```diff
@@ -4116,6 +4145,10 @@ public class TraceJittedMethod
         /// </summary>
         public int NativeSize;
         /// <summary>
+        /// Heap size allocated for JIT code of method
+        /// </summary>
+        public int AllocatedHeapSize;
```

Comment:
Should we name this something like `LoaderHeapAllocSize` to be clear that these are loader heap allocations, instead of C++ heap allocations?  (If so, we should also update the various related fields.)